pages/join/[...joinCode].js

```diff
@@ -0,0 +1,110 @@
+import Head from 'next/head';
+import Navbar from '../../components/navbar';
+import { useState } from 'react';
+import { useRouter } from 'next/router';
+
+export default function JoinWithCode() {
+  const [formData, setFormData] = useState({});
+
+  async function saveEmail(e) {
+    e.preventDefault();
+    const response = await fetch(`/api/student_email_join`, {
+      method: 'PUT',
+      body: JSON.stringify(formData)
+    });
+    alert('Successfully registered, please extit window');
+    return await response.json();
+  }
+
+  const router = useRouter();
+  const { joinCode } = router.query;
+  return (
+    <div>
+      <Head>
+        <title>Create Next App</title>
+        <meta name='description' content='Generated by create next app' />
+        <link rel='icon' href='/favicon.ico' />
+        <Navbar>
+          <li>
+            <p>Welcome</p>
+          </li>
+        </Navbar>
+      </Head>
+      <h1>{joinCode}</h1>
+      <div className='min-h-full flex items-center justify-center py-40 px-4 sm:px-6 lg:px-8'>
+        <div className='max-w-md w-full space-y-8'>
+          <div>
+            <h2 className='mt-6 text-center text-3xl font-extrabold text-gray-900'>
+              Verify with FreeCodeCamp Email Address
+            </h2>
+          </div>
+          <form className='mt-8 space-y-6' onSubmit={saveEmail}>
+            <input type='hidden' name='remember' value='true'></input>
+            <div className='rounded-md shadow-sm -space-y-px'>
+              <div>
+                <label htmlFor='email-address' className='sr-only'>
+                  Email address
+                </label>
+                <input
+                  onChange={e =>
+                    setFormData({
+                      ...formData,
+                      email: e.target.value,
+                      classId: joinCode
+                    })
+                  }
+                  id='email-address'
+                  name='email'
+                  type='email'
+                  autoComplete='email'
+                  required
+                  className='appearance-none rounded-none relative block w-full px-3 py-2 border border-gray-300 placeholder-gray-500 text-gray-900 rounded-t-md focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 focus:z-10 sm:text-sm'
+                  placeholder='Email address'
+                ></input>
+              </div>
+            </div>
+
+            <div className='flex items-center justify-between'>
+              <div className='text-sm'>
+                <a
+                  href='https://www.freecodecamp.org/'
+                  className='font-medium text-indigo-600 hover:text-indigo-500'
+                >
+                  {' '}
+                  Register on FreeCodeCamp{' '}
```

Comment:
```suggestion
                  Register on freeCodeCamp{' '}
```

@freeCodeCamp/classroom the freeCodeCamp is always styled as the f small, and the C's are capital.

Reply:
Fixed the freeCodeCamp text